dependencies.yaml

```diff
@@ -46,14 +47,14 @@ files:
     includes:
       - build_all
       - libarrow_run
-      - cudatoolkit
+      - cuda_version
```

Comment:
Small request: let's alphabetize this section while we're here (`cuda_version` before `libarrow_run`). Same for `test_notebooks` and other dependency lists.

Reply:
sure! Just pushed https://github.com/rapidsai/cudf/pull/14733/commits/1b306cf81e83bc27ac5ba4d534056a272e64d40f alphabetizing the sections that contain a `- cuda_version`.

Reply:
sure! Just pushed https://github.com/rapidsai/cudf/pull/14733/commits/1b306cf81e83bc27ac5ba4d534056a272e64d40f alphabetizing the sections that contain a `- cuda_version`.